docs/walkthrough-1.6.md

```diff
@@ -277,9 +277,9 @@ ups-instance-credential           Opaque                                2
 
 Notice that a new `Secret` named `ups-instance-credential` has been created.
 
-# Step 6 - Unbinding from the ServiceInstance
+# Step 6 - Deleting the `ServiceInstanceCredential`
 
-Now, let's unbind from the instance.  To do this, we simply *delete* the
+Now, let's unbind from the provisioned instance. To do this, we simply *delete* the
```

Comment:
seems inconsistent to use 'delete' in the title but 'unbind' in the text.  I'm ok with either one, but we should be consistent

Reply:
@duglin this document tries to equate Kubernetes actions (delete in this case) with the equivalent OSB concept (unbind in this case), so I kept with that trend.

Reply:
as a newbie I would be confused by the switch in verbs, perhaps do something like:
`Now, let's unbind from (delete) the provisioned instance`

Reply:
no that's not right either....

Reply:
skip my comment - we can tweak it in a follow-on PR

Reply:
@duglin if you'd like to switch the verbs around, I'm happy to do that. It'll require a change in the rest of the doc too, so how about we do that in a follow-up?